frontend/module/yace-editor.js

```diff
@@ -467,14 +467,14 @@ const builtins = new Set([
   "rand",
   "rand1",
   "move",
-  "read",
```

Comment:
read is still a builtin, isn't it?

Reply:
yeah, it was a duplicate. I'll break it out into a separate commit (ie should be deleted).